content/en/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md

```diff
@@ -168,14 +168,36 @@ The kubelet provides a gRPC service to enable discovery of dynamic resources of
 running Pods. For more information on the gRPC endpoints, see the
 [resource allocation reporting](/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins/#monitoring-device-plugin-resources).
 
-## Limitations
-
-The scheduler plugin must be involved in scheduling Pods which use
-ResourceClaims. Bypassing the scheduler by setting the `nodeName` field leads
-to Pods that the kubelet refuses to start because the ResourceClaims are not
-reserved or not even allocated. It may be possible to [remove this
-limitation](https://github.com/kubernetes/kubernetes/issues/114005) in the
-future.
+## Pre-scheduled Pods
+
+When creating a Pod with `nodeName` already set, the scheduler gets bypassed.
+If some ResourceClaim needed by that Pod does not exist yet, is not allocated
+or not reserved for the Pod, then the kubelet will fail to run the Pod and
```

Comment:
it will be great to add instruction on how to unstuck it. I think simply deleting the pod will do. The only thing - mention that if Pod is part of the replicaset, another instance may be created with the same issue.

Reply:
There are numerous reasons for what might be wrong, so listing all possible remediation here will not be possible. The goal in this section is more about raising awareness of the problem ("don't do this"!) and explain that some automatic mitigation is available now through the kube-controller-manager changes.